skyportal/facility_apis/sedm.py

```diff
@@ -100,7 +102,7 @@ def convert_request_to_sedm(request, method_value='new'):
     }
 
     if rtype == "Mix 'n Match":
-        choices = request.payload['observation_choices']
+        choices = deepcopy(request.payload['observation_choices'])
```

Comment:
Maybe make a note why this was needed?

Reply:
Good point, will do!

Yeah it's just that we remove the IFU from that variable few lines after, and since this editing the variable directly then the notification stuff doesn't have it anymore. Hence the deepcopy